### PR TITLE
Vapoursynth

### PR DESCRIPTION
**Pull request by rnhmjoj**
###### Motivation for this change

Add packages for vapoursynth and other dependencies and enable vapoursynth scripting support in mpv. The true motivation is having 24Hz -> 60Hz motion frame interpolation.
###### Things done
- [x] Tested using sandboxing
  ([nix.useChroot](http://nixos.org/nixos/manual/options.html#opt-nix.useChroot) on NixOS,
    or option `build-use-chroot` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [x] NixOS
  - [ ] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change
- [x] Tested execution of all binary files
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by joachifm** [comment]
cc @Fuuzetsu

File: pkgs/development/libraries/zimg/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    owner = "sekrit-twc";
+    repo  = "zimg";
+    rev    = "e88b156fdd6d5ae647bfc68a30e86d14f214764d";
+    sha256 = "1hb35pm9ykdyhg71drd59yy29d154m2r1mr8ikyzpi3knanjn23a";
+  };
+
+  buildInputs = [ autoreconfHook ];
+
+  meta = with stdenv.lib; {
+    description = "Scaling, colorspace conversion and dithering library.";
```
Comment (joachifm) [inline pkgs/development/libraries/zimg/default.nix]: Please remove trailing punctuation from `meta.description`

**Comment by joachifm** [comment]
Looks like `zimg` fails on darwin

**Comment by rnhmjoj** [comment]
Yep, it was reported upstream: https://github.com/sekrit-twc/zimg/issues/52
What do I do?

**Comment by joachifm** [comment]
You could simply disable the darwin build for zimg for the time being. Seeing as mpv is only built on linux currently it shouldn't matter.

Will adding vapoursynth support greatly enlarge the mpv closure?

**Comment by rnhmjoj** [comment]
Quite, It's 479MB larger. Should I disable vapoursynth by default?

**Comment by joachifm** [comment]
Ouch, that's a bit much for my tastes. Is it something most people would want? If not, perhaps it is best to leave it disabled by default.

**Comment by rnhmjoj** [comment]
Well, I think most people doesn't even know it does exist. It's not enabled in other distro distros by default or it has separate package so I'll disable it.

**Pull request merged**